subsys/net/lib/conn_mgr/conn_mgr.h

```diff
@@ -49,4 +51,41 @@ enum net_conn_mgr_state {
 
 void conn_mgr_init_events_handler(void);
 
+/**
+ * @brief Bring all network interfaces admin-up.
+ *
+ * @retval 0 if all calls to net_if_up succeed.
+ * @retval otherwise, the error code returned by the first failed call to net_if_up.
+ */
+int net_conn_mgr_all_if_up(void);
```

Comment:
This one brings all network interface up, would we need a variant where user could say that he/she wants to bring only some parts of the interface up (like some network type etc). Or a variant where it is enough to bring just one interface up. In this case we could add here some parameters to this function.